man/busctl.xml

```diff
@@ -78,10 +78,10 @@
         <term><command>capture</command> <arg choice="opt" rep="repeat"><replaceable>SERVICE</replaceable></arg></term>
 
         <listitem><para>Similar to <command>monitor</command> but
-        writes the output in pcap format (for details, see the <ulink
-        url="https://wiki.wireshark.org/Development/LibpcapFileFormat">Libpcap
-        File Format</ulink> description). Make sure to redirect
-        standard output to a file. Tools like
+        writes the output in pcapng format (for details, see the RFC
+        <ulink url="https://www.ietf.org/staging/draft-tuexen-opsawg-pcapng-02.html">
+        PCAP Next Generation (pcapng) Capture File Format</ulink>).
+        Make sure to redirect standard output to a file. Tools like
```

Comment:
Why "Make sure to redirect standard output to a file."??

Reply:
That was there before.
Since it is binary data, it shouldn't go to terminal.